### ISSUE TITLE
sample toc in it's own design document

### ISSUE THREAD

**Issue by lpatiny** [cheminfo/roc-eln-config#3]
Also in eln.epfl.ch

**Comment by targos** [comment]
If we make this change, we must coordinate the update of the Home views where sampleToc is used

**Comment by stropitek** [comment]
@targos why? The api is based only on view name, not design document name

**Comment by targos** [comment]
OK my bad. Then we can just change it in the config?

**Comment by stropitek** [comment]
Yes that should remove it from one design document and add it to the other.
